ocs/ocp.py

```diff
@@ -134,23 +148,48 @@ def apply(self, yaml_file):
         command = f"apply -f {yaml_file}"
         return self.exec_oc_cmd(command)
 
-    def wait_for_resource_status(
-        self, resource_name, condition, timeout=30, sleep=3
+    def wait(
+        self, condition, resource_name='', selector=None, resource_count=0,
+        timeout=60, sleep=3
     ):
         """
         Wait for a resource to reach to a desired condition
 
         Args:
-            resource_name (str): The name of the resource to wait
-                for (e.g.my-pv1)
             condition (str): The desired state the resource should be at
                 This is referring to: status.phase presented in the resource
                 yaml file
                 (e.g. status.phase == Available)
+            resource_name (str): The name of the resource to wait
+                for (e.g.my-pv1)
```

Comment:
mention default values in docstring (default: def_value), for all below.

Reply:
do we really need that?
a python func with parameter and '=some_value' is enough to understand imo
WDYT?

Reply:
actually don't mind, I just was using this as we mentioned it in our [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md)